src/EditorFeatures/Test/Extensions/SourceTextContainerExtensionsTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.CodeAnalysis.Text;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.UnitTests.Extensions
+{
+    public class SourceTextContainerExtensionsTests
+    {
+        [Fact]
+        public void GetBufferTextFromNonTextContainerThrows()
+        {
+            var containerMock = new Mock<SourceTextContainer>();
+            Assert.Throws<ArgumentException>(containerMock.Object.GetTextBuffer);
+        }
+
+        [Fact]
+        public void GetBufferTextFromTextContainerReturnsTextBuffer()
```

Comment:
What is this actually testing? Is it trying to test that the creation of the text buffer container does not mutate the text buffer, or just that the GetTextBuffer method works at all?